lib/components/user/places/favorite-place-list.js

```diff
@@ -12,9 +12,10 @@ import {
   getPlaceMainText
 } from '../../../util/user'
 import { isBlank } from '../../../util/ui'
+import { Plus } from '@styled-icons/fa-solid'
```

Comment:
```suggestion
import { Plus } from '@styled-icons/fa-solid/Plus'
```
Help! Our bundle size!